src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -323,6 +324,28 @@ public function xdebugAction(): Response
         return new Response($xdebugInfo, 200, ['Content-Type' => 'text/html']);
     }
 
+    /**
+     * Downloads the custom web fonts used in the profiler.
+     *
+     * @throws NotFoundHttpException
+     */
+    public function downloadFontAction(string $fontName): Response
```

Comment:
```suggestion
    public function getFontAction(string $fontName): Response
```

I find `download` misleading as this action does not download anything.

Reply:
perhaps just "load" is enough